example/SINK.xprof/my_callbacks.cpp

```diff
@@ -39,7 +39,7 @@ static void bernard(struct common_data_s *common_data, void *usr_data, int32_t i
     data->count += 1;
 }
 
-void btx_register_usr_callbacks(name_to_dispatcher_t** name_to_dispatcher) {
-  btx_register_callbacks_lttng_ust_ze_profiling_event_profiling_results(name_to_dispatcher, &roger);
-  btx_register_callbacks_lttng_ust_ze_profiling_event_profiling_results(name_to_dispatcher, &bernard);
+void btx_register_usr_callbacks(btx_handle_t** btx_handle) {
```

Comment:
I see, you are using the same replacement "btx_handle" for `common_data` and `name_to_dispatch`. I understand your complain, but here we are mixind two things that even the user would not know and does not care, however, if somebody try to undestand why two different things end up with the same type_name, is that conceptually right?.

Reply:
It's an implementation detail. The user should not care. 
Maybe one day we will but `name_to_dispatcher` into `common_data`. Better to just hide it.